cmd/certs.go

```diff
@@ -24,11 +24,6 @@ import (
 	"path/filepath"
 )
 
-// isSSL - returns true with both cert and key exists.
-func isSSL() bool {
-	return isFile(getPublicCertFile()) && isFile(getPrivateKeyFile())
```

Comment:
> if isFile(getPublicCertFile()) && isFile(getPrivateKeyFile()) {
> 		var err error
> 		globalPublicCerts, err = parsePublicCertFile(getPublicCertFile())
> 		fatalIf(err, "Unable to parse public certificate file %s", getPublicCertFile())
> 
> 		globalRootCAs, err = getRootCAs(getCADir())
> 		fatalIf(err, "Unable to load CA files in %s directory", getCADir())
> 
> 		globalIsSSL = true
> 	}

Perhaps this can be moved into a more enableSSL() function?